Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json

```diff
@@ -0,0 +1,526 @@
+                        "fieldId": "incident_type",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_severity",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_sourcebrand",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_sourceinstance",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_eventtype",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_eventid",
+                        "isVisible": true
+                    }
+                ],
+                "isVisible": true,
+                "name": "Incident Information",
+                "query": null,
+                "queryType": "",
+                "readOnly": false,
+                "type": ""
+            },
+            {
+                "description": "Compromised Card Details",
+                "fields": [
+                    {
+                        "fieldId": "incident_cybleeventsv2cardno",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_cybleeventsv2cardcvv",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_cybleeventsv2cardexpiry",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_cybleeventsv2cardbrand",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_cybleeventsv2cardtype",
+                        "isVisible": true
+                    },
+                    {
+                        "fieldId": "incident_cybleeventsv2cardlevel",
+                        "isVisible": true
+                    }
+                ],
+                "filters": [
+                    [
+                        {
+                            "left": {
+                                "isContext": true,
+                                "value": {
+                                    "simple": "eventtype"
+                                }
+                            },
+                            "operator": "isEqualString",
+                            "right": {
+                                "isContext": false,
+                                "value": {
+                                    "simple": "compromised_cards"
+                                }
+                            },
+                            "type": "shortText"
+                        }
+                    ]
+                ],
```

Comment:
Hi @RosenbergYehuda,
As per your suggestion, I have updated all the files and fixed all the issues that were coming in `ci/circleci: Run Validations` section

However, I am still getting this error

```
[2023-10-18 15:59:47,043] [ERROR] Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json: [ST111] - The field "filters" in path 'quickView'-> 'sections'-> 'Compromised Card Details' was not defined in the scheme
[2023-10-18 15:59:47,043] [INFO] To fix the problem, try running `demisto-sdk format -i Packs/CybleEventsV2/Layouts/layoutscontainer-CybleEventsv2-Layout.json`
```

I was able to fix this too, but I had to remove the `filter` key from the layout as highlighted:
If I remove this part, then the test is passing.

But I believe this filter is required to check if the coming incident is of this particular type: 'compromised_card'. So removing this filter does not make sense. But keeping it is not letting the test pass.
I have exported this layout json from XSOAR instance.

Please suggest how we can go about this.

Reply:
Good job solving most of validations!
@melamedbn , can you please assist here?